cranelift-simplejit/src/backend.rs

```diff
@@ -87,24 +87,34 @@ impl SimpleJITBuilder {
     /// back to a platform-specific search (this typically involves searching
     /// the current process for public symbols, followed by searching the
     /// platform's C runtime).
-    pub fn symbol<K>(&mut self, name: K, ptr: *const u8) -> &Self
+    ///
+    /// # Panics
+    ///
+    /// Panics when ptr is NULL.
+    #[inline]
+    pub unsafe fn symbol<K>(&mut self, name: K, ptr: *const u8) -> &Self
     where
         K: Into<String>,
     {
+        assert!(!ptr.is_null());
         self.symbols.insert(name.into(), ptr);
         self
     }
 
     /// Define multiple symbols in the internal symbol table.
     ///
     /// Using this is equivalent to calling `symbol` on each element.
-    pub fn symbols<It, K>(&mut self, symbols: It) -> &Self
+    ///
+    /// # Panics
+    ///
+    /// Panics when ptr is NULL.
```

Comment:
nit: can you tweak this doc line to tell something like "Panics whenever any symbol pointer is NULL."?

Reply:
Okay, sure thing.